Author: Daniel Baumann <daniel.baumann@progress-technologies.net>
Description:
 Using 'splash' as boot parameter instead of 'plymouth.enable' in system unit
 files (Closes: #680517).

diff -Naurp plymouth.orig/systemd-units/plymouth-halt.service.in plymouth/systemd-units/plymouth-halt.service.in
--- plymouth.orig/systemd-units/plymouth-halt.service.in
+++ plymouth/systemd-units/plymouth-halt.service.in
@@ -3,7 +3,7 @@ Description=Show Plymouth Halt Screen
 After=getty@tty1.service prefdm.service plymouth-start.service
 Before=halt.service
 DefaultDependencies=no
-ConditionKernelCommandLine=!plymouth.enable=0
+ConditionKernelCommandLine=splash
 
 [Service]
 ExecStart=@PLYMOUTH_DAEMON_DIR@/plymouthd --mode=shutdown --attach-to-session
diff -Naurp plymouth.orig/systemd-units/plymouth-kexec.service.in plymouth/systemd-units/plymouth-kexec.service.in
--- plymouth.orig/systemd-units/plymouth-kexec.service.in
+++ plymouth/systemd-units/plymouth-kexec.service.in
@@ -3,7 +3,7 @@ Description=Show Plymouth Reboot with ke
 After=getty@tty1.service prefdm.service plymouth-start.service
 Before=kexec.service
 DefaultDependencies=no
-ConditionKernelCommandLine=!plymouth.enable=0
+ConditionKernelCommandLine=splash
 
 [Service]
 ExecStart=@PLYMOUTH_DAEMON_DIR@/plymouthd --mode=shutdown --attach-to-session
diff -Naurp plymouth.orig/systemd-units/plymouth-poweroff.service.in plymouth/systemd-units/plymouth-poweroff.service.in
--- plymouth.orig/systemd-units/plymouth-poweroff.service.in
+++ plymouth/systemd-units/plymouth-poweroff.service.in
@@ -3,7 +3,7 @@ Description=Show Plymouth Power Off Scre
 After=getty@tty1.service prefdm.service plymouth-start.service
 Before=poweroff.service
 DefaultDependencies=no
-ConditionKernelCommandLine=!plymouth.enable=0
+ConditionKernelCommandLine=splash
 
 [Service]
 ExecStart=@PLYMOUTH_DAEMON_DIR@/plymouthd --mode=shutdown --attach-to-session
diff -Naurp plymouth.orig/systemd-units/plymouth-reboot.service.in plymouth/systemd-units/plymouth-reboot.service.in
--- plymouth.orig/systemd-units/plymouth-reboot.service.in
+++ plymouth/systemd-units/plymouth-reboot.service.in
@@ -3,7 +3,7 @@ Description=Show Plymouth Reboot Screen
 After=getty@tty1.service prefdm.service plymouth-start.service
 Before=reboot.service
 DefaultDependencies=no
-ConditionKernelCommandLine=!plymouth.enable=0
+ConditionKernelCommandLine=splash
 
 [Service]
 ExecStart=@PLYMOUTH_DAEMON_DIR@/plymouthd --mode=shutdown --attach-to-session
diff -Naurp plymouth.orig/systemd-units/plymouth-start.service.in plymouth/systemd-units/plymouth-start.service.in
--- plymouth.orig/systemd-units/plymouth-start.service.in
+++ plymouth/systemd-units/plymouth-start.service.in
@@ -4,7 +4,7 @@ DefaultDependencies=no
 Wants=systemd-ask-password-plymouth.path
 After=systemd-vconsole-setup.service systemd-udev-trigger.service
 Before=systemd-ask-password-plymouth.service
-ConditionKernelCommandLine=!plymouth.enable=0
+ConditionKernelCommandLine=splash
 
 [Service]
 ExecStart=@PLYMOUTH_DAEMON_DIR@/plymouthd --mode=boot --pid-file=@plymouthruntimedir@/pid --attach-to-session
diff -Naurp plymouth.orig/systemd-units/systemd-ask-password-plymouth.path.in plymouth/systemd-units/systemd-ask-password-plymouth.path.in
--- plymouth.orig/systemd-units/systemd-ask-password-plymouth.path.in
+++ plymouth/systemd-units/systemd-ask-password-plymouth.path.in
@@ -5,7 +5,7 @@ DefaultDependencies=no
 Conflicts=shutdown.target
 After=plymouth-start.service
 Before=basic.target shutdown.target
-ConditionKernelCommandLine=!plymouth.enable=0
+ConditionKernelCommandLine=splash
 ConditionPathExists=/run/plymouth/pid
 
 [Path]
diff -Naurp plymouth.orig/systemd-units/systemd-ask-password-plymouth.service.in plymouth/systemd-units/systemd-ask-password-plymouth.service.in
--- plymouth.orig/systemd-units/systemd-ask-password-plymouth.service.in
+++ plymouth/systemd-units/systemd-ask-password-plymouth.service.in
@@ -5,7 +5,7 @@ DefaultDependencies=no
 Conflicts=shutdown.target
 After=plymouth-start.service
 Before=shutdown.target
-ConditionKernelCommandLine=!plymouth.enable=0
+ConditionKernelCommandLine=splash
 ConditionPathExists=/run/plymouth/pid
 
 [Service]
